test/test_xla_dist.py

```diff
@@ -704,6 +721,121 @@ def test_healthy_cluster(self):
         client_master_ip='10.0.0.0')
     self.assertEqual(expected, cluster)
 
+  def test_healthy_remote_coordinator(self):
+    noop_compute_service = build_mock_compute_service({}, {})
+    self.mock_discovery.side_effect = build_mock_services_fn(
+        noop_compute_service)
+
+    tpu_map = {
+        'fake-pod': {
+            'state':
+                'READY',
+            'health':
+                'HEALTHY',
+            'runtime_version':
+                'v2-nightly',
+            'accelerator_type':
+                'v3-32',
+            'api_version':
+                'V2_ALPHA1',
+            'network_endpoints': [{
+                'ipAddress':
+                    f'10.1.0.{index}',
+                'port':
+                    '8470',
+                'greenVmSelflink':
```

Comment:
This should be removed. Internal only.

Reply:
Hmm, for running script on tpuvm case, we actually query the `greenVmSelflink` to get the instance name to build the client worker. Should that also be changed?

Reply:
Yes as that's an internal field. Syncing offline.